Extension/package.nls.json

```diff
@@ -12,6 +12,7 @@
     "c_cpp.command.resetDatabase.title": "Reset IntelliSense Database",
     "c_cpp.command.takeSurvey.title": "Take Survey",
     "c_cpp.command.buildAndDebugActiveFile.title": "Build and Debug Active File",
+    "c_cpp.command.restartLanguageServerActiveFile.title": "Restart Language Server on Active File",
```

Comment:
"Language Server" references should be renamed "IntelliSense" or possibly "IntelliSense Server" or "IntelliSense Process".

Reply:
In Visual Studio, the menu item is simply "Rescan File".  I think the use of "Active" in this menu item (and the one above it) could be dropped.

Reply:
The potential problem with removing the "active" part is that in other context such as the Command Palette it may not as obvious that it's referring to the active file. Also there are a bunch of VS Code "File" commands that refer to the active file.

Also, the term "scan" or "rescan" is not used anywhere else with our extension and it's not really explicitly saying what exactly is being done. Also, "scanning" in the context of text processing usually refers to a quick O(N) operation and not a long parsing operation.

Reply:
IMHO, we should remove the "Active" part.  Perhaps word it as "ForFile"/"for file", to indicate it doesn't set it for all files.  It would be relatively simple to add an optional command argument indicating which file to reset.  That would be extensible, allowing other extension developers to reset IntelliSense for a specific file (though, I'm not sure the user case).

Reply:
Your comment seems to be a reason to not remove the Active part, makes it more confusing in the event that we decide to allow the command to take a file argument, i.e. "Rescan Active File" versus "Rescan File..." (user navigates to the file or it's passed via some arg).

Reply:
![image](https://user-images.githubusercontent.com/19859882/132923511-53849f29-db14-4831-8c3b-4349240a1af6.png)

without the "active", it would be reasonable in this context for the user to mistakenly believe they're allowed to choose the file and not make one active first. In particular, we wanted to avoid users running those commands with the wrong file active.

Reply:
I'm OK with leaving "Active" in the text description that is displayed in the UI, but I think the command itself could so easily be extended, that it should not contain "Active" in the implementation of the code.

I have similar feedback in the native PR, where "Active" is referred to despite the URI being passed in from the TypeScript side.  In that case, since the file is provided to in the message, the code shouldn't refer to the "Active" file.

Reply:
Yeah, that sounds fine.

Reply:
So we can't have the command say "active file" without also having the item in the context menu say "active file"?  I'm fine with leaving "active" in the command where the context is less obvious.  It was the context menu item I was hoping to remove it from because it seems unnecessary.

Reply:
@bobbrow The command has a string description that is shared by all contexts, Command Palette and context menu.

Reply:
That's too bad

Reply:
![image](https://user-images.githubusercontent.com/16143955/133672308-790996ba-f753-4440-8f00-1544bada5e10.png)
This is what the user will see, but the functions itself will send the document info to the language server, so inside the code the name "active" is not used, to keep the function more general.

Reply:
Ok